web-registry/src/components/templates/ProjectDetails/hooks/useOtherProjects.ts

```diff
@@ -1,10 +1,71 @@
-import { useMoreProjectsQuery } from '../../../../generated/graphql';
+import { useMemo } from 'react';
+import { QueryProjectsResponse } from '@regen-network/api/lib/generated/regen/ecocredit/v1/query';
+import _ from 'lodash';
```

Comment:
```suggestion
import shuffle from ‘lodash/shuffle`
```

Lodash is notoriously bad at code splitting 
I was going to suggest
```tsx
import { shuffle } from ‘lodash`
```

as it’s what we’re doing in most of the app and I thought it would be properly tree-shaked, but found [this thread](https://www.blazemeter.com/blog/import-lodash-libraries) which made me curious, and turns out it wasn’t working correctly. 

Made a pr to fix: https://github.com/regen-network/regen-web/pull/1182

Also, why are we randomizing these values? just to make sure it displays differently?

Reply:
Nice, thanks!

yes, exactly, to display a random subset